README.md

```diff
@@ -1,14 +1,43 @@
 ## CDA2FHIR
+![Status](https://img.shields.io/badge/Status-Build%20Passing-lgreen)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
 <img src="./img/img.jpg" alt="img" width="400"/>
 
-Translating Cancer Data Commons (CDA) to FHIR (Fast Healthcare Interoperability Resources) format.
+Translating Cancer Data Commons (CDA) to 🔥 FHIR (Fast Healthcare Interoperability Resources) format.
 
+## Usage 
 ### Installation
 
+- from source 
 ```commandline
 # clone repo & setup virtual env
 python3 -m venv venv
 . venv/bin/activate
 pip install -e .
 ```
```

Comment:
* Add Related projects: section to point to cda aggregator project
* Add a "contact the author" to retrieve the raw data